src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseGradingService.java

```diff
@@ -728,57 +731,41 @@ private boolean createFeedbackForDuplicateTests(Result result, ProgrammingExerci
      * Update the score given the positive tests score divided by all tests' score.
      * Takes weight, bonus multiplier and absolute bonus points into account.
      * All tests in this case do not include ones with visibility=never.
```

Comment:
Improve this comment to explcitly state the the filtering is expected to be done prior and the filtered Set must be passed.

Reply:
I don't know how to word such a comment